src/main/kotlin/org/jitsi/nlj/rtcp/RtcpParsers.kt

```diff
@@ -19,9 +19,14 @@ package org.jitsi.nlj.rtcp
 import org.jitsi.nlj.transform.node.PacketParser
 import org.jitsi.rtp.rtcp.CompoundRtcpPacket
 import org.jitsi.rtp.rtcp.RtcpPacket
+import org.jitsi.utils.logging2.Logger
 
-class CompoundRtcpParser : PacketParser("Compound RTCP parser", {
-    CompoundRtcpPacket(it.buffer, it.offset, it.length) })
+class CompoundRtcpParser(parentLogger: Logger) : PacketParser("Compound RTCP parser", parentLogger, {
+    CompoundRtcpPacket(it.buffer, it.offset, it.length).also { compoundPacket ->
+        // Force packets to be evaluated to trigger any parsing errors
+        compoundPacket.packets
```

Comment:
Is there another case where we want the lazy init of `compoundPacket.packets`? If not, let's just remove the `lazy`

Reply:
I thought, logically, it made sense to keep it that way in the packet (I think we've done this more on the semantics of the fields rather than checking all the current usages).

Reply:
I find it unexpected. All other packets parse in place.